app/lib/child_tax_credit_calculator.rb

```diff
@@ -1,15 +1,18 @@
 class ChildTaxCreditCalculator
   PER_DEPENDENT_UNDER_SIX_PAYMENT = 300
-  PER_DEPENDENT_OVER_SIX_PAYMENT = 250
+  PER_DEPENDENT_SIX_AND_OVER_PAYMENT = 250
 
-  def self.monthly_payment_due(dependents_under_six_count:, dependents_over_six_count:)
+  def self.monthly_payment_due(dependents)
+    dependents_under_six_count = dependents.count { |dependent| dependent.age_at_end_of_year(2021) < 6 }
+    dependents_six_and_over_count = dependents.count - dependents_under_six_count
     dependents_under_six_payment = dependents_under_six_count * PER_DEPENDENT_UNDER_SIX_PAYMENT
-    dependents_over_six_payment = dependents_over_six_count * PER_DEPENDENT_OVER_SIX_PAYMENT
+    dependents_six_and_over_payment = dependents_six_and_over_count * PER_DEPENDENT_SIX_AND_OVER_PAYMENT
 
-    dependents_under_six_payment + dependents_over_six_payment
+    dependents_under_six_payment + dependents_six_and_over_payment
   end
 
-  def self.total_payment_due(dependents_under_six_count:, dependents_over_six_count:)
-    monthly_payment_due(dependents_under_six_count: dependents_under_six_count, dependents_over_six_count: dependents_over_six_count) * 12
+  # This is half of the total CTC due for 2021, but clients have to file in 2022 for the 2nd half
+  def self.total_advance_payment_2021(dependents)
+    monthly_payment_due(dependents) * 6
```

Comment:
One additional thing -- Since we're filing 2020 tax returns, even though this is advance payment they're getting in 2021 it might be more consistent with naming conventions to use the tax year instead of the filing year here.